users_ldap_populate/models/users_ldap.py

```diff
@@ -123,9 +123,10 @@ def get_ldap_entry_dicts(self, conf, user_name='*', timeout=60):
         """
         ldap_filter = filter_format(conf['ldap_filter'] % user_name, ())
         conn = self.connect(conf)
-        conn.simple_bind_s(conf['ldap_binddn'] or '',
-                           conf['ldap_password'] or '')
-        results = conn.search_st(conf['ldap_base'], ldap.SCOPE_SUBTREE,
+        ldap_password = conf['ldap_password'] or ''
+        ldap_binddn = conf['ldap_binddn'] or ''
+        conn.simple_bind_s(ldap_binddn.encode('utf-8'), ldap_password.encode('utf-8'))
```

Comment:
Another travis error, sorry for missing that on previous comment. Lines 128 and 129 are too long.

Best to have function arguments on separate lines with indent of four spaces. Most readable...